### ISSUE TITLE
Issue with nuisance suggestion overwriting </div>

### ISSUE THREAD

**Issue by bobbyconnolly** [codota/TabNine#411]
**please complete the following information:**	
 - OS version: [e.g. Win 10, Mac Catalina]	Win10
 - Editor: [e.g. vscode, intelij, sublime]	vscode
 - Editor version:	1.57.1
 - Programming language: [e.g. js, ts, c++]	html / react.js
 - TabNine extension version: 3.4.10
 - Engine version (could be located within Tabnine Hub):  3.5.5-40

**Issue Details:**

What happens is I type `<div>` and get to here:
![image](https://user-images.githubusercontent.com/29126769/123521610-c213bd00-d685-11eb-8d98-419b188f3fdc.png)

so far so good

Then I press Enter and this happens:
![image](https://user-images.githubusercontent.com/29126769/123521595-ab6d6600-d685-11eb-97bf-91e286071943.png)

What I was hoping for was
![image](https://user-images.githubusercontent.com/29126769/123521630-e2dc1280-d685-11eb-98ef-2c1bbc69d666.png)

I realize that Tabnine is picking up on things I did previously. However, this issue happens to me over and over when doing the render/return of a react component. 

Basically, what I need is to have the default response to Enter keypress be nothing. Tabnine made itself the default and this completely breaks my dev experience 😭 .

Otherwise I love the product. 

Thanks for your help!

gz#2470

**Comment by Thomasalex2** [comment]
I have been having these same issues ever since I started working on HTML files. Every single time I open tags and click on enter it replaces the closing tag with whatever it was suggested. 

Please train the models such a way that it does not override the closing tag, or at the very least give us the option to use some alternative key like Tab to accept suggestions
